hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -276,18 +276,18 @@
       tests: ["flash_ctrl_mid_op_rst"]
     }
     {
-      name: ecc
+      name: double_bit_err
       desc: '''
-            Randomly enable ECC for a randomly selected set of pages. Randomly corrupt a single bit
-            in the memory that is about to be read and ensure that the ECC works - the corrupted
-            bit should be fixed. Corrupt randomly either the data or the ECC bits. Randomly corrupt
-            2 bits in the same word and ensure that the read results in error. Ensure that pages
-            with ECC not enabled reads back corrupted data without any errors. Verify both types,
-            pre-scramble ECC(integrity ECC, 4-bits) and post-scramble ECC(reliability ECC, 8-bits).
-	    Test status and control ECC bits.
+            Run read / write test and inject double bit error randomly for read transactions. -
+            both direct and controller read -.
+            Check op_status.err and err_code.rd_err are asserted for ctrl read and
+            tlul response error for host read.
+            Check fatal alert is adderted for reliability ecc errors (double bits) and
```

Comment:
adderted? I do not know what this word means :)

Reply:
fixed typo.